client/blocks/taxonomy-manager/list.jsx

```diff
@@ -176,15 +162,20 @@ export class TaxonomyManagerList extends Component {
 	}
 }
 
+const getTermIds = ( terms ) => ( Array.isArray( terms ) ? terms.map( ( term ) => term.ID ) : [] );
```

Comment:
Not exactly sure how I feel about this. Should it be its own selector? It felt wrong to me constructing `termIds` on the class instance using lifecycle methods.